SPECS/libtiff/libtiff.spec

```diff
@@ -1,13 +1,14 @@
 Summary:        TIFF libraries and associated utilities.
 Name:           libtiff
 Version:        4.4.0
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        libtiff
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Libraries
 URL:            https://gitlab.com/libtiff/libtiff
 Source0:        https://gitlab.com/libtiff/libtiff/-/archive/v%{version}/libtiff-v%{version}.tar.gz
+Patch0:         CVE-2022-2056_CVE-2022-2057_CVE-2022-2058.patch
```

Comment:
I don't think that will work correctly.
It should be a single patch file per patch.

Reply:
It's more about the scan tooling than the change

Reply:
We can check with Nick.

Alternate option is to similar to what we discussed yesterday.  Create one patch file and then consider 3 indpendent nopatch files.

Reply:
Got it. I can make that change.